src/components/FavoriteArticles/FavoriteArticles.js

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment } from 'react';
+import styles from '../UserProfileTabs/UserProfileTabs.module.scss';
+import apple from '../../assets/images/apple.jpg';
+import microsoft from '../../assets/images/microsoft.png';
+
+const FavoriteArticles = () => (
+  <Fragment>
+    <a href="/favorites">
+      <div className={ styles.articlesGrid }>
+        <div>
+          <img src={ apple } alt="andela" className={ styles.articleImage }/>
+        </div>
+        <div className={ styles.articleContents }>
+          <div>
+            <h1> Apple Technology </h1>
+          </div>
+          <div>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. 
+              Sed cum totam, ex porro cupiditate explicabo deleniti fugit 
+              reprehenderit aliquid soluta repellendus rem magnam. 
+              Dolor nesciunt harum, perspiciatis est nemo id sequi nam, 
+            </p>
+          </div>
+          <div className={ styles.summaryFooter }>
+            <div>
```

Comment:
I see you are having `<div>` with empty attributes. is there any reason for that?

Reply:
I used the div to group two elements together (that is the paragraph for the author's name and span for article creation date and read time). When I don't have that div, they seem to be far apart

Reply:
Yeah, that's cool. But I see you are using it for an element like `<h1>` which is a block element

Reply:
Okay. I will correct this